src/components/CommandBar/CommandBarReview.tsx

```diff
@@ -57,19 +60,71 @@ function CommandBarReview({ stepBack }: { stepBack: () => void }) {
     })
   }
 
+  const availableOptionalArgs = useMemo(() => {
+    if (!selectedCommand?.args) return undefined
+    const s = { ...selectedCommand.args }
+    for (const [name, arg] of Object.entries(s)) {
+      const value =
+        (typeof argumentsToSubmit[name] === 'function'
+          ? argumentsToSubmit[name](commandBarState.context)
+          : argumentsToSubmit[name]) || ''
+      const isHidden =
+        typeof arg.hidden === 'function'
+          ? arg.hidden(commandBarState.context)
+          : arg.hidden
+      const isRequired =
+        typeof arg.required === 'function'
+          ? arg.required(commandBarState.context)
+          : arg.required
+      if (isHidden || isRequired || value) {
+        delete s[name]
+      }
+    }
+    return s
+  }, [selectedCommand, argumentsToSubmit, commandBarState.context])
   return (
-    <CommandBarHeader>
-      <p className="px-4 pb-2">
-        {selectedCommand?.reviewMessage ? (
-          selectedCommand.reviewMessage instanceof Function ? (
-            selectedCommand.reviewMessage(commandBarState.context)
-          ) : (
-            selectedCommand.reviewMessage
-          )
-        ) : (
-          <>Confirm {selectedCommand?.displayName || selectedCommand?.name}</>
```

Comment:
@franknoirot btw this is no more, it felt weird to have this default review message with the new layout

Reply:
Yeah that makes sense, the buttons are now clear enough to know you're about to submit anyway so we don't need this crutch